### PR TITLE
[CAC-2024] DevopsDays Cáceres - add speakers and sponsors

### PR DESCRIPTION
**Pull request by neovasili**
This PR adds latest confirmed sponsors and speakers.

### CODE REVIEW

File: static/events/2024-caceres/speakers/diego-duran.jpg
Comment (nimbinatus) [inline static/events/2024-caceres/speakers/diego-duran.jpg:1]: These files are a bit big. Can you resize to make the filesizes smaller? 600px by 600px usually does it; looking for the files to be under 1MB or else the site loads very slowly.

File: static/events/2024-caceres/speakers/rui-zhu.jpg
Comment (nimbinatus) [inline static/events/2024-caceres/speakers/rui-zhu.jpg:1]: Resize, please :)

**Review by nimbinatus — CHANGES_REQUESTED** [review]
Hi there! Can you resize a couple of the images?

File: static/events/2024-caceres/speakers/diego-duran.jpg
Comment (neovasili) [reply static/events/2024-caceres/speakers/diego-duran.jpg:1]: resized!

File: static/events/2024-caceres/speakers/rui-zhu.jpg
Comment (neovasili) [reply static/events/2024-caceres/speakers/rui-zhu.jpg:1]: resized!

**Pull request merged**